Makefile

```diff
@@ -46,6 +46,13 @@ build.bench:
 build.examples.as:
 	@cd ./examples/assemblyscript/testdata && npm install && npm run build
 
+.PHONY: build.examples.zig
+build.examples.zig: examples/allocation/zig/testdata/greet.wasm
+
+%.wasm: %.zig
+	@(cd $(@D); zig build)
+	@mv $(@D)/zig-out/bin/$(@F) $(@D)
```

Comment:
Can we add the zig-out/** into .gitignore?

Reply:
I added zig-out and zig-cache in my local so that I would not add those dirs accidentally, but didn't push it as I was not sure you like that. Done.
https://github.com/tetratelabs/wazero/pull/741/commits/c2aafcf362573739e76eade10b66459713eb1390